guides/v2.2/release-notes/release-candidate/install.md

```diff
@@ -44,8 +44,8 @@ There are three Magento code repositories on GitHub where you can find Release C
   </tr>
 <tr>
     <td><b>Magento CE</b></td>
-    <td><a href="https://github.com/magento/magento2">https://github.com/magento/magento2</a></td>
-    <td>develop</td>
+    <td><a href="https://github.com/magento/magento2ce">https://github.com/magento/magento2ce</a></td>
```

Comment:
This is technically correct, but the `magento2ce/` repo is private and requires special access permissions. Instead, change the public **magento/magento2** branch from `develop` to `2.2.0-preview`.